### PR TITLE
[L2B-5425] Add root level attributes

### PR DESCRIPTION
**Pull request by maciekop-l2b**

### CODE REVIEW

File: packages/backend/src/modules/tracked-txs/TrackedTxsModule.ts [hunk 1]
```diff
@@ -91,7 +91,7 @@ export function createTrackedTxsModule(
     const coingeckoClient = providers.coingeckoClient
     const coingeckoQueryService = new CoingeckoQueryService(
       coingeckoClient,
-      logger.tag('trackedTxs'),
+      logger.tag({ tag: 'trackedTxs' }),
```
Comment (antooni) [inline packages/backend/src/modules/tracked-txs/TrackedTxsModule.ts:94]: feature: trackedTxs

File: packages/backend/src/modules/activity/indexers/BlockTargetIndexer.ts [hunk 2]
```diff
@@ -11,9 +11,15 @@ export class BlockTargetIndexer extends RootIndexer {
     private readonly blockTimestampProvider: BlockTimestampProvider,
     projectId: ProjectId,
   ) {
-    super(logger.tag(projectId), {
-      tickRetryStrategy: Indexer.getInfiniteRetryStrategy(),
-    })
+    super(
+      logger.tag({
+        tag: projectId,
```
Comment (antooni) [inline packages/backend/src/modules/activity/indexers/BlockTargetIndexer.ts:16]: what about chain?

File: packages/backend/src/modules/tracked-txs/TrackedTxsModule.ts (hunk 1 above)
Comment (maciekop-l2b) [reply packages/backend/src/modules/tracked-txs/TrackedTxsModule.ts:96]: trackedTxs is not a feature, added as a module

**Review by antooni — APPROVED** [review]

File: packages/backend/src/modules/activity/indexers/DayActivityIndexer.ts [hunk 3]
```diff
@@ -8,7 +8,10 @@ export class DayActivityIndexer extends ManagedChildIndexer {
     super({
       ...$,
       name: `activity_day_indexer`,
-      tag: $.projectId,
+      tags: {
+        tag: $.projectId,
```
[the remark this answers is not in the document]
Comment (maciekop-l2b) [reply packages/backend/src/modules/activity/indexers/DayActivityIndexer.ts:12]: we will ad it later if needed

**Pull request merged**